README.md

```diff
@@ -44,6 +44,18 @@ from numcompress import compress, decompress
 >>> decompress('Acn[rpB{n@')
 [145.78, 127.6, 135.26]
 
+# compressing and decompressing numpy arrays
+
+>>> from numcompress import compress_ndarray, decompress_ndarray
+>>> import numpy as np
+
+>>> series = np.random.randint(1, 100, 100).reshape(10, 10)
+
+>>> compress_ndarray(series)
```

Comment:
Let us take a small series (say 5 by 5) and show pre, post, and compressed text version of it. Casual reader of Readme would also get a clear idea of what our library offers.

Something like this -
<img width="722" alt="Screenshot 2020-08-31 at 4 30 56 PM" src="https://user-images.githubusercontent.com/1350490/91713613-b8294680-eba7-11ea-95cc-f66cfaf70b61.png">